Documentation.app/Contents/MacOS/Generation examples.playground/Pages/Limitations.xcplaygroundpage/Contents.swift

```diff
@@ -7,4 +7,15 @@
 /*:
  # Limitations
  
+ Bow OpenAPI is very powerful and generates a ready to use network client in Swift that is as good as the specification you provide. If the specification is poor or flaky, the generated code will have the same problems. Generating code according to this specification is a way of honoring it, but it is important that this contract is also honored by the backend side.
+ 
+ Nevertheless, Bow OpenAPI has some other limitations.
+ 
+ ## Inline data types
+ 
+ In order to have a proper generation, you need to define data models in the `components` section in OpenAPI, or in the `definitions` section in Swagger. Defining your data models inline can result in generated code that is not properly named and therefore difficult to use.
```

Comment:
nested models: inline nested models are not supported - you need to define the nested model as a ref to `components` or `definitions` section.